### PR TITLE
Verify MAC Multicast

### PR DESCRIPTION
**Pull request by axkar**
This PR enhances the existing "static-multicast-filters" test by adding verification for MAC multicast functionality.

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [x] Test changes
  - [x] Checked in changed Readme.adoc (make test-spec)
  - [ ] Added new test to group Readme.adoc and yaml file
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
  - [ ] ChangeLog updated (for major changes)
- [ ] Other (please describe):

### CODE REVIEW

File: test/infamy/multicast.py [hunk 1]
```diff
@@ -54,3 +55,29 @@ def __exit__(self, _, __, ___):
...
+        
+class MacMCastSender:
+    def __init__(self, netns, group):
+        self.group = group
+        self.netns = netns
+
+    def __enter__(self):
+        send_cmd = (
+            "from scapy.all import sendp, Ether; "
+            f"pkt=Ether(src='d2:68:c1:eb:e4:26', dst='{self.group}', type=0xdead); "
```
Comment (mattiaswal) [inline test/infamy/multicast.py:67]: Do you required to specify src? If not it feel strange to have random hard coded address, should it not be the one of the exiting interface?

Comment (axkar) [reply test/infamy/multicast.py:67]: Thanks. It's leftover from debugging. I'll remove it :)

**Review by mattiaswal — APPROVED** [review]
Great work (:+1: )

File: test/infamy/multicast.py (hunk 1 above)
Comment (axkar) [reply test/infamy/multicast.py]: Changed to something that does not look like an actual MAC address, since scapy doesn't seem to allow avoiding src addr

**Pull request merged**